csuntechlab/iSTART
# STR-101 -- FE: Make dashboard page mobile responsive

# Description
Made desktop version responsive; added specific alt-text depending on substance.
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. View the mobile and desktop viewports and ensure everything scales properly.
3. Check to see if there is any overflows, positioning of elements.
4. Ensure it meets the format of the desktop mock-up.
